hazelcast-jet-kafka/src/test/java/com/hazelcast/jet/impl/connector/kafka/StreamKafkaPTest.java

```diff
@@ -94,6 +150,111 @@ public void run() throws Exception {
         assertTrueEventually(() -> assertTrue(jobFuture.isDone()));
     }
 
+    @Test
+    public void when_snapshotSaved_then_offsetsRestored() throws Exception {
+        StreamKafkaP[] processor = {new StreamKafkaP(properties, singletonList(topic1Name), 1, 0, 60000)};
+        TestOutbox[] outbox = {new TestOutbox(new int[] {10}, 10)};
```

Comment:
I can't make out a reason why these two are packed into an array.

Reply:
They are both modified and used in lambdas.